fashion-hyperparameter/optimize.py

```diff
@@ -0,0 +1,210 @@
+import csv
+import os
+
+from pathlib import Path
+
+from jina.flow import Flow
+from jina.helloworld.helper import download_data
+from jina.proto import jina_pb2
+
+
+class Hyperoptimizer:
+    def __init__(self):
+        self.parameters = [
+            {
+                'JINA_INDEX_DATA_FILE': 'tests/hyperparameter/index_data.csv',
+                'JINA_EVALUATION_DATA_FILE': 'tests/hyperparameter/query_data.csv',
+                'JINA_WORKSPACE': 'workspace_eval',
+                'JINA_MATRIX_SIZE': size,
+            }
+            for size in range(10, 100)
+        ]
+
+    def parameters_iterator(self):
+        yield {}, self.save_results
```

Comment:
it should not get the next `parameter`? how does this work?

Reply:
Please ignore this code for now. We are still implementing and haven't deleted the parts, which were just sketches. This draft is for us to see better what is already added.